docker/hooks/build

```diff
@@ -11,6 +11,10 @@ echo "||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
 ./dev-build-images.sh "$DOCKER_REPO/"
 
+echo "#### pushing images ####"
+./dev-build-images.sh "$DOCKER_REPO/"
```

Comment:
Is this a typo? Shouldn't this  be `dev-push-images`?

Reply:
same comment as below, isn't it? Should be fixed, thanks!